app/serializers/v1/credit_serializer.rb

```diff
@@ -13,6 +13,10 @@ def serialize
           code: model.item_code,
           name: model.item_name,
         },
+        invoice: {
+          lago_id: model.invoice_id,
+          payment_status: model.invoice.payment_status,
+        }
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.